docs/installing_hypershift_cli.md

```diff
@@ -6,23 +6,24 @@ The Hosted Control Plane command-line interface (CLI) is an `oc` command-line [p
 
 ## Installing the Hosted Control Plane CLI
 
-**NOTE: This document is unofficial will be updated later. For now download the CLI binary from this page
+1. [Enable the Hypershift feature](https://github.com/stolostron/hypershift-addon-operator/blob/main/docs/provision_hosted_cluster_on_mce_local_cluster.md#configuring-the-hosting-service-cluster)
 
-1. Download the HyperShift CLI binary.
+2. On the OCP console, click on the ? button at the top right hand corner and choose `Command line tools` menu to download the HyperShift CLI binary.
 
+(unofficial: you can download one of these as well)
 - [HyperShift CLI binary for Linux x86_64](https://github.com/stolostron/hypershift-addon-operator/blob/main/docs/hypershift-linux-amd64.tar.gz?raw=true)
 - [HyperShift CLI binary for Mac x86_64](https://github.com/stolostron/hypershift-addon-operator/blob/main/docs/hypershift-darwin-amd64.tar.gz?raw=true)
 - [HyperShift CLI binary for Windows x86_64](https://github.com/stolostron/hypershift-addon-operator/blob/main/docs/hypershift-windows-amd64.tar.gz?raw=true)
 
-2. Unpack the archive.
+3. Unpack the archive.
 
 ```
     $ tar xvzf <file>
 ```
 
-3. Rename the file to `oc-hcp` to make it an `oc` command-line [plugin](https://docs.openshift.com/container-platform/4.11/cli_reference/openshift_cli/extending-cli-plugins.html). `hcp` stands for `Hosted Control Plane`.
+4. Rename the file to `oc-hcp` to make it an `oc` command-line [plugin](https://docs.openshift.com/container-platform/4.11/cli_reference/openshift_cli/extending-cli-plugins.html). `hcp` stands for `Hosted Control Plane`.
```

Comment:
Would we not say rename hypershift to oc-hcp? or offer
```
#Linux
sudo mv ./hypershift /usr/bin
```
Which probably exists on most linux and macOs systems?

Reply:
@jnpacker Currently the binaries are built with different names like hypershift-linux-amd64 in the same folder. I should change the CICD docker file to generate binary file `hypershift` in different folders like `linux/amd64`. Once this change is made, I will make the doc change.

Reply:
✔️ACK